openedx/features/pakx/lms/overrides/utils.py

```diff
@@ -55,6 +55,12 @@ def _get_org_log(organization):
     return org_logo.url if org_logo else org_logo
 
 
+def is_blank_str(value):
+    if value.isspace() or not value:
```

Comment:
can we refactor this to something like this?
`return value.isspace() or not value`

Reply:
sure